docs/_guides/codewind-crw-quick-guide.md

```diff
@@ -0,0 +1,206 @@
+* Develop a simple microservice that uses Eclipse Codewind on CodeReady Workspaces
+
+## Overview
+Use Eclipse Codewind to develop microservice applications from application stacks in an integrated developer environment (IDE). CodeReady Workspaces provides a containerized IDE for cloud native application development on an OpenShift cluster. 
+
+## Developing with CodeReady Workspaces 
+CodeReady Workspaces uses Kubernetes and containers to provide a preconfigured environment. Use CodeReady Workspaces to create, build, and test your code in OpenShift containers but feel like you are working on an IDE on your local computer.  
+
+### Prerequisite
+CodeReady Workspaces requires at least two 1Gi ReadWriteOnce (RWO) persistent volumes on the cluster to install and a 1Gi RWO volume for each created workspace.
+
+Each Codewind workspace also requires at least on 1Gi ReadWriteMany (RWX) persistent volume.
+
+### Installing CodeReady Workspaces
+1. Subscribe to the CodeReady Workspaces operator then install it.
+  * To manually install CodeReady Workspaces, apply the CodeReady Workspaces OperatorHub subscription. 
+2. After the CodeReady Workspaces operator is installed, create the CheCluster resources. 
+  * If you have not set up an auth provider, set `openShiftoAuth` to `false`. 
+  * To enable HTTPS, set `tlsSupport` to `true`.
+  * If you use a self-signed certificate, set `selfSignedCert` to `true`. 
+3. If you install Codewind alongside CodeReady Workspaces, use the following yaml:  
+```yaml
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+metadata:
+  name: codeready-workspaces
+spec:
+  server:
+    cheImageTag: ''
+    cheFlavor: codeready
+    devfileRegistryImage: ''
+    pluginRegistryImage: ''
+    tlsSupport: false
+    selfSignedCert: false
+    cheWorkspaceClusterRole: 'eclipse-codewind'
+    customCheProperties:
+      CHE_INFRA_KUBERNETES_WORKSPACE__START__TIMEOUT__MIN: "15"
+      CHE_LIMITS_WORKSPACE_IDLE_TIMEOUT: "0"
+      CHE_WORKSPACE_PLUGIN__BROKER_WAIT__TIMEOUT__MIN: "15"
+  database:
+    externalDb: false
+    chePostgresHostName: ''
+    chePostgresPort: ''
+    chePostgresUser: ''
+    chePostgresPassword: ''
+    chePostgresDb: ''
+  auth:
+    openShiftoAuth: false
+    identityProviderImage: ''
+    externalIdentityProvider: false
+    identityProviderURL: ''
+    identityProviderRealm: ''
+    identityProviderClientId: ''
+  storage:
+    pvcStrategy: per-workspace
+    pvcClaimSize: 1Gi
+    preCreateSubPaths: true
+```
+  * To manually install Codewind alongside Codewind Workspaces, save this yaml to a disk and run `oc apply -f <yaml>`. 
+4. CodeReady Workspaces now installs and you can access the CodeReady Workspaces URL and Red Hat SSO Admin Console URL.  
+
+### Setting up Codewind
+Before you can use Codewind with CodeReady Workspaces or Che, complete the following steps, either before or after you install Codewind: 
+
+1. Apply the Codewind cluster role with `oc apply -f`. 
```

Comment:
I think this should be a prerequisite step before installing CRW

Reply:
On this note, do we need to include additional instructions to install Codewind? Like a link to our existing instructions to install Codewind?